Get basic information for a page, shorthand for getting page likes.

asolove opened the pull request:
A small feature I needed for a project I'm working on.

I'm brand new to Elixir, so happy to receive any feedback on code style, etc.

mweibel commented on lib/facebook.ex:
```diff
@@ -114,6 +114,43 @@ defmodule Facebook do
+	@spec page(page_id :: integer | String.t, access_token, fields, options) :: response
+	def page(page_id, access_token, fields, options) do
+		params = [fields: fields]
+		if !is_nil(access_token) do
```
do you need that check? can you fetch the page without having an access_token?

mweibel commented on lib/facebook.ex:
```diff
@@ -114,6 +114,43 @@ defmodule Facebook do
+	"""
+	@spec page(page_id :: integer | String.t) :: response
+	def page(page_id) do
+		page(page_id, nil, [], [])
```
probably here you should also accept an access_token, but if the access_token is not required (see other comment), we can leave it

asolove commented:
You're exactly right so I've removed the unnecessary function definition and nil check. Also added a few smaller items that are not directly related to the PR but would be nice.

mweibel commented:
Neat :) thanks. Published version 0.3.0
